drivers/spi/spi_mcux_flexio.c

```diff
@@ -100,6 +101,11 @@ static int spi_mcux_flexio_isr(void *user_data)
 	const struct spi_mcux_flexio_config *config = dev->config;
 	struct spi_mcux_flexio_data *data = dev->data;
 
+#if defined(CONFIG_SOC_SERIES_KE1XZ)
+	/* Wait until data transfer complete. */
+	WAIT_FOR((0U == (FLEXIO_SPI_GetStatusFlags(config->flexio_spi)
```

Comment:
I would also like to know if this applies to other platforms

Reply:
To be on the safe side, I think this macro can be retained and then modified as needed when porting and testing this driver for other platforms in the future.